### PR TITLE
Feat cart

### PR DESCRIPTION
**Pull request by humzahasan**
# Description

Added shopping cart pages for ecom website

## Type of change

- [ ] Bug fix
- [X] New feature
- [ ] Breaking change
- [ ] Documentation updated

## Screenshot

screenshot

### CODE REVIEW

File: pages/Wishlist/wishlist.html [hunk 1]
```diff
@@ -78,15 +78,11 @@ <h1 class="md-title">My Wishlist</h1>
           </div>
           <div class="card-media">
             <img src="../../assets/pasta.jpg" alt="" class="card-img" />
```
Comment (zoef-yt) [inline pages/Wishlist/wishlist.html:80]: Changes: Could you fill the alt attribute

File: pages/Wishlist/wishlist.html [hunk 2]
```diff
@@ -110,15 +106,11 @@ <h1 class="md-title">My Wishlist</h1>
...
```
Comment (zoef-yt) [inline pages/Wishlist/wishlist.html:108]: Changes: Could you fill the alt attribute

File: pages/Wishlist/wishlist.html [hunk 3]
```diff
@@ -142,15 +134,11 @@ <h1 class="md-title">My Wishlist</h1>
...
```
Comment (zoef-yt) [inline pages/Wishlist/wishlist.html:136]: Changes: Could you fill the alt attribute

File: pages/Cart/cart.html [hunk 4]
```diff
@@ -61,5 +61,117 @@
...
+                <i class="fas fa-heart"></i>Remove from Cart
+              </button>
+              <button class="btn btn-icon">
+                <i class="fas fa-share"></i>Move To Wishlist
+              </button>
+            </div>
+          </div>
+          <div class="card card-horizontal-lg">
+            <div class="card-media">
+              <img src="../../assets/pancake.jpg" alt="" class="card-img" />
```
Comment (zoef-yt) [inline pages/Cart/cart.html:96]: Changes: Could you fill the alt attribute

File: pages/Cart/cart.html [hunk 5]
```diff
@@ -61,5 +61,117 @@
         </div>
       </nav>
     </header>
+    <main class="flex-center-column">
+      <h1 class="md-title">My Shopping Cart</h1>
+      <div class="grid-col-2 grid-col-7by3">
+        <div class="grid-item">
+          <div class="card card-horizontal-lg">
+            <div class="card-media">
+              <img src="../../assets/burger.jpg" alt="" class="card-img" />
```
Comment (zoef-yt) [inline pages/Cart/cart.html:70]: Changes: Could you fill the alt attribute

File: pages/Cart/cart.html [hunk 6]
```diff
@@ -61,5 +61,117 @@
...
+            </div>
+            <div class="card-text">
+              <p class="card-title">Veg Burger</p>
+              <p class="card-subtitle">Rs.160</p>
+              <div class="input-counter">
+                <button class="btn btn-primary">
+                  <a href="#btnaction"> +</a>
+                </button>
+                <input type="number" name="" id="" value="5" />
```
Comment (zoef-yt) [inline pages/Cart/cart.html:79]: Suggestion: name and id are empty it can be removed.

File: pages/Cart/cart.html [hunk 7]
```diff
@@ -61,5 +61,117 @@
...
+              <img src="../../assets/pancake.jpg" alt="" class="card-img" />
+            </div>
+            <div class="card-text">
+              <p class="card-title">Banana Pancake</p>
+              <p class="card-subtitle">Rs.230</p>
+              <div class="input-counter">
+                <button class="btn btn-primary">
+                  <a href="#btnaction"> +</a>
+                </button>
+                <input type="number" name="" id="" value="2" />
```
Comment (zoef-yt) [inline pages/Cart/cart.html:105]: Suggestion: name and id are empty it can be removed.

File: pages/Cart/cart.html [hunk 8]
```diff
@@ -61,5 +61,117 @@
...
+                <i class="fas fa-heart"></i>Remove from Cart
+              </button>
+              <button class="btn btn-icon">
+                <i class="fas fa-share"></i>Move To Wishlist
+              </button>
+            </div>
+          </div>
+          <div class="card card-horizontal-lg">
+            <div class="card-media">
+              <img src="../../assets/salad.jpg" alt="" class="card-img" />
```
Comment (zoef-yt) [inline pages/Cart/cart.html:122]: Changes: Could you fill the alt attribute

File: pages/Cart/cart.html [hunk 9]
```diff
@@ -61,5 +61,117 @@
...
+            </div>
+            <div class="card-text">
+              <p class="card-title">Salad</p>
+              <p class="card-subtitle">Rs.100</p>
+              <div class="input-counter">
+                <button class="btn btn-primary">
+                  <a href="#btnaction"> +</a>
+                </button>
+                <input type="number" name="" id="" value="10" />
```
Comment (zoef-yt) [inline pages/Cart/cart.html:131]: Suggestion: name and id are empty it can be removed.

**Review by zoef-yt — CHANGES_REQUESTED** [review]
The code looks good. The UI is great. Suggestions can be added. 👍

File: pages/Cart/cart.html (hunk 7 above)
Comment (sudipkundu999) [inline pages/Cart/cart.html:105]: remove empty attributes

**Review by sudipkundu999 — APPROVED** [review]
UI looks clean and so is the code.
Good to merge with minor fix

**Comment by humzahasan** [comment]
@zoef-yt & @sudipkundu999 merging this PR. 

Thank you for the feedback, catering the same in the next PR.

https://github.com/humzahasan/Hunger-Pangs/pull/6

**Pull request merged**